Makefile.org

```diff
@@ -424,6 +424,14 @@ clean:	libclean
 	rm -fr $$i/*; \
 	done
 
+distclean: clean
+	-$(RM) `find . -type l -a \! -path "./.git/*"`
+	$(RM) apps/CA.pl
+	$(RM) test/evptests.txt test/newkey.pem test/testkey.pem test/testreq.pem
+	$(RM) tools/c_rehash
+	$(RM) crypto/opensslconf.h
+	$(RM) Makefile Makefile.bak
```

Comment:
If one goes for recursive it might be appropriate to use the opportunity and add `*.S` to `clean:` rules. There was report about `*.S` not being cleaned up [on ARM]...